src/doc/trpl/while-loops.md

```diff
@@ -88,6 +88,18 @@ for x in 0..10 {
 }
 ```
 
+You may also encounter situations where you have nested loops and need to specify which one your `break` or `continue` statement is for. Like most other languages, by default a `break` or `continue` will apply to innermost loop. In a sitation where you would like to a `break` or `continue` for one of the outer loops, you can use labels to specify which loop the `break` or `continue` statement applies to. This will only print when both x and y are odd:
```

Comment:
can you wrap this line please? Also, `x` and `y` need to be in graves for highlighting